app/models/tag.rb

```diff
@@ -1,6 +1,6 @@
 class Tag < ActiveRecord::Base
   belongs_to :user
   has_and_belongs_to_many :posts
-
   validates_presence_of :user, :title
+  validates :title, uniqueness: true
```

Comment:
We added this validation just for testing, I'm not sure it should be there. What do you think @FerPerales ?